lib/git_repository/cloning.rb

```diff
@@ -75,7 +77,7 @@ def is_git_repository?(address)
     end
 
     def svn_ls(address)
-      exec 'svn', 'ls', address
+      exec('svn', 'ls', address)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.